sdk/package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@0xsequence/abi": "^1.4.3",
     "@0xsequence/core": "^1.4.3",
+    "@0xsquid/sdk": "^2.8.24",
```

Comment:
not sure how this diff ended up here. Maybe a weird merge with main? Its tagged the SDK team for review because of it I think

Reply:
After running through the steps to rebuild the SDK, these things popped in 🤷